### PR TITLE
Remove request cert in favor https options

### PR DESCRIPTION
**Pull request by evilebottnawi**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [x] This is a **code refactor**
- [x] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?

Yes

### Motivation / Use-Case

Avoid unnecessary option

### Breaking Changes

Yes

### Additional Info

No

### CODE REVIEW

File: lib/Server.js [hunk 1]
```diff
@@ -573,6 +573,8 @@ class Server {
 
       this.options.https.key = this.options.https.key || fakeCert;
       this.options.https.cert = this.options.https.cert || fakeCert;
+
+      console.log(this.options.https);
```
Comment (hiroppy) [inline lib/Server.js:577]: Need to delete this line.

Comment (evilebottnawi) [reply lib/Server.js:577]: Thanks!

File: lib/options.json [hunk 2]
```diff
@@ -296,8 +296,8 @@
     "public": {
       "type": "string"
     },
-    "requestCert": {
-      "type": "boolean"
+    "publicPath": {
```
Comment (hiroppy) [inline lib/options.json:299]: why does this pr have publicPath changed?

Comment (evilebottnawi) [reply lib/options.json]: @hiroppy merge problem, fixed

**Comment by evilebottnawi** [comment]
@hiroppy no, we already have `this.options.https.requestCert`, it is boolean value, so we don't need to read it from filesystem

**Comment by hiroppy** [comment]
oh, sorry my misreading.

**Pull request merged**